jupiterone/util.go

```diff
@@ -39,3 +41,20 @@ func jsonDiffSuppressFunc(k, oldValue, newValue string, d *schema.ResourceData)
 
 	return reflect.DeepEqual(old, new)
 }
+
+func queriesToMapStringInterface(c []client.QuestionQuery) ([]map[string]interface{}, error) {
+	buf := new(bytes.Buffer)
+	enc := json.NewEncoder(buf)
+	enc.SetEscapeHTML(false)
+	err := enc.Encode(c)
+	if err != nil {
+		return nil, err
+	}
+
+	msi := []map[string]interface{}{}
+	err = json.Unmarshal(buf.Bytes(), &msi)
+	if err != nil {
+		return nil, err
+	}
+	return msi, nil
+}
```

Comment:
Terraform can't diff the changes in user defined types like `QuestionQuery` very well or at all. Changing to the generic `map[string]interface{}` allow TF to read changes and compare them.